tests/cisco_ios/show_logging/cisco_ios_show_logging.raw

```diff
@@ -0,0 +1,3 @@
+Nov  4 15:37:33.641: %LINK-3-UPDOWN: Interface GigabitEthernet1/0/12, changed state to up
```

Comment:
Do you mind adding quite a few more lines from the output? We're thinking of some use cases of one line overflowing to the next for the same log message, etc.